aws/resource_aws_instance_test.go

```diff
@@ -32,6 +34,12 @@ func init() {
 	})
 }
 
+func testAccErrorCheckSkipEC2(t *testing.T) resource.ErrorCheckFunc {
+	return testAccErrorCheckSkipMessagesContaining(t,
+		"VolumeTypeNotAvailableInRegion: Instance launch can not be fulfilled",
```

Comment:
Nit: Seems like the error code portion is descriptive enough in case the messaging changes

```suggestion
		"VolumeTypeNotAvailableInRegion",
```